software/ensemblvep/meta.yml

```diff
@@ -0,0 +1,53 @@
+name: ENSEMBLVEP
+description: Ensembl Variant Effect Predictor (VEP)
+keywords:
+    - annotation
+tools:
```

Comment:
Do we document the `params` somewhere here? 

I would find it important to document how the `vep_tag` container magic works and how to use it.

Reply:
that's a good point.
cf https://github.com/nf-core/sarek/pull/386
I have dedicated subworfklows that I plan to add in this repo.
Would you like to have these in this PR and https://github.com/nf-core/modules/pull/546
Or in separates?

Reply:
I think every module should be usable and understandable as a unit, therefore I'm +1 adding it here and in #546. 

Ideally, we would have a `params` key in the `meta.yml` but I have no clue if that has been done before and matches our specification of `meta.yml`.

Reply:
added the params as input.
What do you think of it?

Reply:
added the params